### PR TITLE
Lesson-3-1

### PR DESCRIPTION
**Pull request by XeniyaDob**
Styling using CSS modules

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -94,15 +99,18 @@ const App = () => {
...
-      <h1>Todo list</h1>
-      <Routes>
-        <Route
-          index
-          exact
-          path="/"
-          element={
-            <>
-              <AddTodoForm onAddTodo={addTodo} />
+      <div className={styles.container}>
```
Comment (tsbrun) [inline src/App.js:102]: Remember to add a closing </div> tag for this div

File: src/App.js [hunk 2]
```diff
@@ -94,15 +99,18 @@ const App = () => {
...
+        <Sidebar />
+        <div className={styles.todoContainer}>
```
Comment (tsbrun) [inline src/App.js:104]: These two divs need a closing tags

File: src/App.js [hunk 3]
```diff
@@ -2,6 +2,11 @@ import React, { useState } from "react";
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import styles from "./App.module.css";
+<<<<<<< HEAD
+import Sidebar from "./Sidebar";
+=======
+>>>>>>> 42c54d3d54e4a7961743e7440745f8cd7f66b313
```
Comment (tsbrun) [inline src/App.js:9]: Make sure to remove the following lines from the file:

```
<<<<<<< HEAD
=======
>>>>>>> 42c54d3d54e4a7961743e7440745f8cd7f66b313
```

**Review by tsbrun — CHANGES_REQUESTED** [review]
Looks good. Just some minor changes to make. Great work 👍🏽

File: src/App.js (hunk 3 above)
Comment (XeniyaDob) [reply src/App.js:9]: Yes, I will do it. Thank you :)

**Pull request merged**